engine/src/main/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProvider.java

```diff
@@ -130,4 +129,29 @@ public void initCounters() {
         } while (rbacUsers.getData().size() == rbacElementsPerPage);
         return users;
     }
+
+    private List<User> transformToUser(List<ITUserResponse> itUserResponses, boolean adminsOnly) {
+        List<User> users = new LinkedList<>();
```

Comment:
Why `LinkedList`?

Reply:
Could also use ArrayList or something else. With that amount of entries it probably doesn't matter. Will add ArrayList